reactfire/tsconfig.json

```diff
@@ -10,6 +10,5 @@
     "outDir": "pub/reactfire",
     "declaration": true,
     "skipLibCheck": true
-  },
-  "files": ["index.ts"]
+  }
```

Comment:
I should .npmignore the test files now that They’re getting built in the pub dir